tests/test_config_parser.py

```diff
@@ -185,14 +190,14 @@ def _append_list(self, ref_entry, new_args, job_name=DEFAULT_JOB_NAME, separator
                 self.reference_config[job_name][ref_entry].append(arg)
 
     @staticmethod
-    def write_ini(dname: str, fname: str, config: dict, prio=1000) -> None:
+    def write_ini(dname: str, fname: str, config: Dict[str, Any], prio: int = 1000) -> None:
```

Comment:
At some point we pass a `Dict[str, object] `, but at other it becomes `Dict[str, Dict[str, str]]`. Went with this approach, let me know if there's a better way.